src/main/java/com/ays/backend/user/model/entities/User.java

```diff
@@ -64,12 +55,32 @@ public class User extends BaseEntity {
     private UserStatus status;
 
     @Column(nullable = false)
-    private int countryCode;
+    private Integer countryCode;
 
     @Column(nullable = false)
-    private int lineNumber;
+    private Integer lineNumber;
 
     @Column
     private LocalDateTime lastLoginDate;
 
+    @Column(name = "organization_id")
+    private Long organizationId;
+
+    public static User from(AdminRegisterRequest registerRequest, PasswordEncoder passwordEncoder) {
+
+        return User.builder()
```

Comment:
Status field is required in the user object, the endpoint doesn't work with an empty status. So please update this method, change the AdminRegisterRequest. And write tests accordingly.

Reply:
I just added status field in from method of User.